versions/v1.0.0/Metaverse/Trade/Sell.adoc

```diff
@@ -27,6 +27,25 @@ interface Sell {
 }
 ----
 
+===== Fields
+
+`action` *Required*:: Action of selling metaverse token, e.g. `sell`.
+`address` *Optional*:: Address of the sold metaverse token, e.g. `0x306b1ea3ecdf94aB739F1910bbda052Ed4A9f949`.
+`id` *Optional*:: ID of the sold metaverse token, e.g. `159`.
+`name` *Optional*:: Name of the sold metaverse token, e.g. `Beanz`.
+`standard` *Optional*:: Standard of the sold metaverse token, e.g. `ERC-721`.
+`symbol` *Optional*:: Symbol of the sold metaverse token, e.g. `BEANZ`.
+`value` *Required*:: Amount of the sold metaverse token, e.g. `1`.
+`uri` *Optional*:: URI of the sold metaverse token, e.g. `ipfs://QmdYeDpkVZedk1mkGodjNmF35UNxwafhFLVvsHrWgJoz6A/beanz_metadata/159`.
+`parsed_image_url` *Optional*:: Parsed image URL of the sold metaverse token, e.g. `https://ipfs.io/ipfs/QmdYeDpkVZedk1mkGodjNmF35UNxwafhFLVvsHrWgJoz6A/beanz_metadata/159`.
+`cost` *Required*:: Cost of the sold metaverse token. Should include the following fields.
+`cost.name` *Optional*:: Name of the cost token, e.g. `Wrapped Ether`.
+`cost.symbol` *Optional*:: Symbol of the cost token, e.g. `WETH`.
+`cost.decimals` *Optional*:: Decimal places of the cost token, e.g. `18`.
```

Comment:
Different terms to describe the same concepts ( e.g., decimals of the cost token  and decimal places of the cost token)

Reply:
It's defined in node project using same Token struct.